music_album.rb

```diff
@@ -0,0 +1,25 @@
+require './item'
+
+# MusicAlbum class for music album object creation.
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  @music_albums = []
+  def initialize(on_spotify:, **attributes)
+    super(attributes)
+    @on_spotify = on_spotify
+    self.class.all << self
+  end
+
+  def can_be_archived?
+    Date.today.year - @publish_date.year > 10 && @on_spotify == true
+  end
```

Comment:
- [ ] As it was pointed out earlier, in order to avoid code repetition, since `can_be_archived` had been implemented in the `superclass` (item.rb); kindly use the super keyword to access the method. 👍 This will show a good comprehension of Inheritance in Ruby.

Reply:
Done